src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
+    type--footer();
+    align-items: center;
     background: $color--greyscale--white;
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+    
+    // Remove from the document flow and hide. Can't use display: none due to
+    // https://github.com/tj/nib/issues/291
+    position: absolute;
+    visibility: hidden;
+    +mq--show-footer() {
```

Comment:
not a fan of ```mq```. it's not clear what it stands for (media query?) and even then i'm not sure what media-query--show-footer would mean